scripts/pages/map-selector/map-selector.ts

```diff
@@ -2,6 +2,8 @@ import { OnPanelLoad, PanelHandler } from 'util/module-helpers';
 import { traverseChildren } from 'util/functions';
 import { MapCreditType } from 'common/web';
 
+const DASHBOARD_URL = 'https://dashboard.momentum-mod.org';
+
```

Comment:
Yeah, I noticed this, and was doing in the second PR. Since afaik this hasn't been reviewed much and I have a second PR worth of stuff that's very related, I'm going to merge them together and fixup some commits so this is clearer.